### ISSUE TITLE
configure colors/themes

### ISSUE THREAD

**Issue by sandro-pasquali** [grigorii-horos/cli-html#3]
Hi, great lib.

Any way to set the theme config for `ansi-colors` (https://github.com/doowb/ansi-colors#themes)?

It would be great to be able to control the color schemes for output.

**Comment by grigorii-horos** [comment]
Yes — theming is fully supported now (the library has changed a lot since this issue).

Three levels of customization:

1. **Global theme file** — drop a `config.yaml` in your config dir to override any default:
   - Linux: `~/.config/cli-html/config.yaml` (or `~/.config/cli-markdown/config.yaml` for the markdown CLI)
   - macOS: `~/Library/Preferences/cli-html/config.yaml`
   - Windows: `%LOCALAPPDATA%\\cli-html\\Config\\config.yaml`

2. **Per-call theme (library)** — pass a theme object:
   ```js
   import { renderHTML } from 'cli-html';
   renderHTML('<h1>Hi</h1>', { h1: 'red bold', a: { color: 'cyan underline' } });
   ```

3. **Per-element (inline)** — `data-cli-*` attributes on individual HTML elements.

Colors use chalk-string syntax (e.g. `"red bold"`, `"bgBlue white underline"`). See `config.yaml` in the repo for the full list of themeable elements and the README for details.

Closing as resolved — feel free to reopen if something specific is missing.
